client_types.go

```diff
@@ -84,6 +84,7 @@ type TunnelState struct {
 	IP               string   `json:"ip_address,omitempty"`
 	Metadata         Metadata `json:"metadata,omitempty"`
 	Owner            string   `json:"owner"`
+	Shared           string   `json:"shared,omitempty"`
```

Comment:
Looks like TunnelState could be also versioned.

Reply:
agreed, once we get to a stable API, we'd need to version the spec.